contents/docs/experiments/frequentist-method.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Frequentist statistical method
+---
+
+> You can set the default statistical method for experiments in **Settings > Organization > General**. You can also override this on a per-experiment basis in each experiment's settings.
+
+PostHog supports two statistical approaches: **Bayesian** and **Frequentist**. This page explains how our **Frequentist** engine works, what metrics it supports, and how to interpret its results.
+
+Frequentist testing is the classical statistics approach used in most academic and industry-standard A/B testing tools. Instead of assigning probabilities to outcomes, it asks: _“If there were no real difference between variants, how likely would we be to observe results like this?”_ That’s the foundation for the p-value and confidence interval you'll see in experiment results.
+
+## Supported metric types
+
+We support two metric types for Frequentist analysis:
+
+### Funnel
+
+Calculates the percentage of users for whom the metric occurred at least once (e.g. "Signed up", "Clicked CTA"). These are binary outcomes: the event either happened for a user or it didn’t.
+
+### Mean
+
+Tracks the value of the metric per user (e.g. revenue, session duration, number of clicks). These are continuous values, and typically include more variation per user.
```

Comment:
Count of clicks is not continous
```suggestion
Tracks the value of the metric per user (e.g. revenue, session duration, number of clicks). These are numerical values and can often vary significantly between users.
```
Also, maybe add a link to "Outlier handling" in the new engine docs here?